extensions/nanoarrow_ipc/CMakeLists.txt

```diff
@@ -16,9 +16,13 @@
 # under the License.
 
 message(STATUS "Building using CMake version: ${CMAKE_VERSION}")
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.14)
 include(FetchContent)
 
+if(NOT DEFINED CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
```

Comment:
Fun fact, this actually does not enforce the usage of C++11 but allows a compiler to decay to an earlier version.
See https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD_REQUIRED.html